### PR TITLE
Lesson2_Task1

### PR DESCRIPTION
**Pull request by stalgatuly**
1) Print a welcoming message.
2) Print the first 10 Fibonacci numbers by importing a package. 
3) Print a complete ending message using defer keyword.

### CODE REVIEW

**Review by denislituev — COMMENTED** [review]
This commit with first task shouldn't be here in PR with the second task

File: lesson_02/fibonacci/fibonacci.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package fibonacci
+
+import "fmt"
+
+func Print_fib_loop(n int) {
```
Comment (denislituev) [inline lesson_02/fibonacci/fibonacci.go]: good naming style for functions in go is **camelCase**, not **snake_case**

File: lesson_02/fibonacci/fibonacci.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	for count := 1; count <= n; count++ {
+		fmt.Printf("%d ", a)
+		temp := a
+		a = b
+		b += temp
+	}
+	fmt.Println()
+}
+
+func Print_fib_recursion(n int) {
```
Comment (denislituev) [inline lesson_02/fibonacci/fibonacci.go]: same here with function naming

File: lesson_02/fibonacci/fibonacci.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	for count := 1; count <= n; count++ {
+		fmt.Printf("%d ", fib_nth(count))
+	}
+	fmt.Println()
+}
+
+func fib_nth(nth int) int {
```
Comment (denislituev) [inline lesson_02/fibonacci/fibonacci.go]: same here with function naming

**Review by denislituev — CHANGES_REQUESTED** [review]

**Review by nikonok — APPROVED** [review]

**Review by denislituev — APPROVED** [review]

**Pull request merged**